server/apps/publish/content/common.py

```diff
@@ -79,8 +77,9 @@ class BasePublishService(BaseService):
     publish_type = 'publish'
     published_state = 'published'
 
-    non_digital = partial(filter, lambda s: s.get('subscriber_type', '') != SUBSCRIBER_TYPES.DIGITAL)
-    digital = partial(filter, lambda s: s.get('subscriber_type', '') == SUBSCRIBER_TYPES.DIGITAL)
+    non_digital = partial(filter, lambda s: s.get('subscriber_type', '') == SUBSCRIBER_TYPES.WIRE)
+    digital = partial(filter, lambda s: (s.get('subscriber_type', '') == SUBSCRIBER_TYPES.DIGITAL) or
+                      (s.get('subscriber_type', '') == SUBSCRIBER_TYPES.ALL))
```

Comment:
This lambda function can be simplified to:
```py
lambda s: s.get('subscriber_type') in {SUBSCRIBER_TYPES.DIGITAL, SUBSCRIBER_TYPES.ALL}
```

(assuming that both of the checked SUBSCRIBER_TYPES values are a non-empty string)